# Duckiebot parameter. 
wheel_distance: 0.106         # Wheel distance in [m]. 

# Position of vehicle in camera's frame [m/quat].
# with alpha = tilting angle of camera = 12Â°. 
cam_x: 0.0
cam_y: 0.09088
cam_z: -0.0197
cam_qx: -0.5495232
cam_qy: 0.5495232
cam_qz: -0.4449991
cam_qw: -0.4449991

# Differential vehicle model - Process noise. 
process_noise_x: 0.1        # Variance of x estimate [m].  
process_noise_y: 0.1        # Variance of y estimate [m].  
process_noise_t: 0.1        # Variance of thetha estimate [rad].           

# April tag localization - Measurement noise. 
april_noise_x: 0.01           # Variance of x estimate [m].  
april_noise_y: 0.01           # Variance of y estimate [m].  
april_noise_t: 0.01           # Variance of thetha estimate [rad].      
